frontend/public/locales/en/translation.json

```
@@ -148,12 +148,12 @@
     "important": "Important",
     "crucial": "Crucial",
     "criteria": "Criteria",
-    "today": "Today",
-    "thisWeek": "This week",
-    "thisMonth": "This month",
-    "thisYear": "This year",
-    "allTime": "All time",
-    "uploadDate": "Upload date",
+    "today": "A day",
+    "thisWeek": "A week",
+    "thisMonth": "A month",
+    "thisYear": "A year",
+    "allTime": "Always",
+    "uploadDate": "Upload since",
```

I think it will be clearer this way:

```suggestion
    "uploadDate": "Uploaded since",
```